tests/kernel/interrupt/testcase.yaml

```diff
@@ -38,7 +38,11 @@ tests:
       - CONFIG_MINIMAL_LIBC=y
   arch.shared_interrupt:
     # excluded because of failures during test_prevent_interruption
-    platform_exclude: qemu_cortex_m0
+    platform_exclude:
```

Comment:
Nit: now these two comments are not placed correctly:
* `# excluded because of failures during test_prevent_interruption`
* `# same as above, #22956`

I'd suggest to move the first comment directly above the `qemu_cortex_m0` entry in the `platform_exclude` list, and modify the second comment to say `# same as for qemu_cortex_m0, #22956`.

Reply:
Done :)

Reply:
Thanks! LGTM now :)

I'll ACK the PR when the CI finishes running and doesn't report any errors